src/settings_gui.cpp

```diff
@@ -2187,12 +2187,14 @@ struct GameSettingsWindow : Window {
 		} else {
 			/* Only open editbox if clicked for the second time, and only for types where it is sensible for. */
 			if (this->last_clicked == pe && sd->desc.cmd != SDT_BOOLX && !(sd->desc.flags & SGF_MULTISTRING)) {
+				int64 value64 = value;
```

Comment:
Question from IRC: `are the int64 correct? or should they be Money?`

I honestly do not know the answer :)

Reply:
Well, first of all, you picked the wrong int64 :p
But in some other places it may have been better to use Money and I even tried only to find out
1) There is no Money * Money 
2) I've no idea how to fix that
3) Other parts of code use int64 for that
So I thought I did enough semi-related fixes for one PR already :p

Reply:
Don't be picky! :P :D Tnx for the explanation, happy to read you did consider it!